<launch>
        <arg name="path" default="/vol/pepper/systems/pepper-robocup-nightly/share/openpose/models/"/>
        <arg name="path_to_config"
            default="/vol/pepper/systems/pepper-robocup-nightly/share/pepper_perception_configs/vision/openpose_ros/shirtcolor.yaml"/>
	<node name="detect_people_server" pkg="openpose_ros" type="detect_people_server"  output="screen" >
                <remap from="/xtion/rgb/image_raw" to="/pepper_robot/sink/front/image_raw"/>
		<param name="net_input_size_width" value="160" />
		<param name="net_input_size_height" value="80" />
		<param name="net_output_size_width" value="160" />
		<param name="net_output_size_height" value="80" />
		<param name="output_size_width" value="640" />
		<param name="output_size_height" value="480" />
		<param name="scale_number" value="1" />
                <param name="scale_gap" value="0.3" />
                <param name="extended_people_topic" value="/people_tracker/people/extended" />
                <param name="person_attribute_service_topic" value="/open_pose/get_person_attributes" />
                <param name="crowd_attribute_service_topic" value="/open_pose/get_crowd_attributes" />
                <param name="image_topic" value="/pepper_robot/sink/front/image_raw" />
                <param name="bottom_image_topic" value="/pepper_robot/sink/bottom/image_raw" />
                <param name="models_folder" value="$(arg path)" />
                <param name="path_to_config" value="$(arg path_to_config)" />
                <param name="gpu_id" value="0" />
                <param name="visualize" value="true" />
		<param name="visualize_uuid" value="false" />
    </node>
</launch>
